akd/src/directory.rs

```diff
@@ -838,7 +854,7 @@ impl<S: Database + 'static, V: VRFKeyStorage> Directory<S, V> {
             .map(|(akd_label, _val)| akd_label.clone())
             .collect();
         // sort the keys, as inserting in primary-key order is more efficient for MySQL
-        keys.sort_by(|a, b| a.cmp(b));
+        keys.sort();
```

Comment:
Actually, do we still need this? This seems to be a database specific optimization, perhaps the sort could be moved into mysql data layer's `get_user_state_versions` (if we want to keep it at all)

Reply:
actually with the transaction ordering and everything i think this is defunct, it was a clippy lint update, but might as well just clean it up. Thanks!